templates/triton_services/README.ipynb

```diff
@@ -0,0 +1,334 @@
+{
```

Comment:
This might be easier to read it from https://github.com/anyscale/templates/blob/b2cf4e6d9c145a815843a7e86190363f2534c1df/templates/triton_services/README.ipynb But this should have the same content ad the `README.md` file